ibis/backends/flink/__init__.py

```diff
@@ -282,12 +310,11 @@ def create_table(
         name
             Name of the new table.
         obj
-            An Ibis table expression or pandas table that will be used to
-            extract the schema and the data of the new table. If not provided,
-            `schema` must be given.
+            An Ibis table expression, pandas or arrow dataframe that will
```

Comment:
going to get nitpicky here :smiley: 


```suggestion
            An Ibis table expression, pandas DataFrame, or PyArrow Table that will
```